src/Microsoft.Data.Analysis/GroupBy.cs

```diff
@@ -37,7 +37,7 @@ public abstract class GroupBy
         /// </summary>
         /// <param name="numberOfRowsInEachGroup"></param>
         /// <returns></returns>
-        public abstract DataFrame Tail(int numberOfRows);
+        public abstract DataFrame Tail(int numberOfRowsInEachGroup);
```

Comment:
From @eerhardt before:

We should update Head as well. Tail and Head should be in sync.